content/en/news/releases/1.12.x/announcing-1.12/upgrade-notes/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.12.0.
+weight: 20
+---
+
+When you upgrade from Istio 1.11.0 to Istio 1.12.0, you need to consider the changes on this page.
```

Comment:
Since we also allow upgrading from 1.10 to 1.12, do we need to change this line and the next?